packages/react-components/react-nav-preview/src/components/NavDrawerHeaderNav/NavDrawerHeaderNav.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
```

Comment:
I don't like that "Nav" is repeated twice in this name. But it's intended to recompose the DrawerHeaderNavigation component. Open to other ideas and suggestions.